### PR TITLE
Splash Screen

### PR DESCRIPTION
**Pull request by MaxH2424**

### CODE REVIEW

File: HomeActivity.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package com.example.foodrecommended_asd;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.graphics.Color;
+import android.os.Bundle;
+import android.view.View;
+
+import gr.net.maroulis.library.EasySplashScreen;
```
Comment (ga0451) [inline HomeActivity.java]: Fix layout of imports

File: HomeActivity.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+import gr.net.maroulis.library.EasySplashScreen;
+
+public class HomeActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home);
+
```
Comment (ga0451) [inline HomeActivity.java]: Add some comments so others can understand.

**Review by ga0451 — COMMENTED** [review]
Simple fixes to make this more readable.

File: HomeActivity.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        EasySplashScreen config = new EasySplashScreen(HomeActivity.this)
+                .withFullScreen()
+                .withTargetActivity(MainActivity.class)
+                .withSplashTimeOut(5000)
+                .withBackgroundColor(Color.parseColor("#1a1b29"))
```
Comment (zacktaylor12359) [inline HomeActivity.java:20]: Consider using a color that is closer to the app's blue background, consider using #00ccff

**Pull request merged**